lib/mps/MPS.jl

```diff
@@ -9,6 +9,7 @@ refer to the [official Apple documentation](https://developer.apple.com/document
 module MPS
 
 using ..Metal
+using Metal.MTL
```

Comment:
```suggestion
using .MTL
```

That should be sufficient, no?